components/application-mgt/org.wso2.carbon.device.application.mgt.api/src/main/java/org/wso2/carbon/device/application/mgt/api/services/impl/CommentManagementAPIImpl.java

```diff
@@ -0,0 +1,228 @@
+        }
+    }
+
+    @Override
+    @DELETE
+    @Path("/{commentId}")
+    public Response deleteComment(
+            @PathParam("commentId") int commentId) {
+
+        CommentsManager commentsManager = APIUtil.getCommentsManager();
+        try {
+            commentsManager.deleteComment(commentId);
+        } catch (CommentManagementException e) {
+            String msg = "Error occurred while deleting the comment.";
+            log.error(msg, e);
+            return Response.status(Response.Status.INTERNAL_SERVER_ERROR)
+                    .entity("Comment with Comment Id" + commentId + " Internal server error occurs").build();
+        } catch (NotFoundException e) {
+            log.error("Not found exception occurs to comment id " + commentId + " .", e);
+            return Response.status(Response.Status.NOT_FOUND).entity("Comment with" + commentId + " not found").build();
+        }
+        return Response.status(Response.Status.OK).entity("Comment is deleted successfully.").build();
+    }
+
+    @Override
+    @GET
+    @Path("/{uuid}")
+    public Response getStars(
+            @PathParam("uuid") String uuid) {
+
+        CommentsManager commentsManager = APIUtil.getCommentsManager();
+        int Stars = 0;
+        try {
+            Stars = commentsManager.getStars(uuid);
+        } catch (CommentManagementException e) {
+            log.error("Comment Management Exception occurs", e);
+            return Response.status(Response.Status.INTERNAL_SERVER_ERROR).build();
+        } catch (ApplicationManagementException e) {
+            log.error("Application Management Exception occurs", e);
+            return Response.status(Response.Status.INTERNAL_SERVER_ERROR)
+                    .entity("Application with UUID" + uuid + " Internal server error occurs").build();
+        }
+        return Response.status(Response.Status.OK).entity(Stars).build();
+    }
+
+    @Override
+    @GET
+    @Path("/{uuid}")
+    public Response getRatedUser(
+            @PathParam("uuid") String uuid) {
+
+        CommentsManager commentsManager = APIUtil.getCommentsManager();
+        int ratedUsers = 0;
+        try {
+            ratedUsers = commentsManager.getRatedUser(uuid);
+        } catch (CommentManagementException e) {
+            log.error("Comment Management Exception occurs", e);
+            return Response.status(Response.Status.INTERNAL_SERVER_ERROR)
+                    .entity("Application with UUID" + uuid + " Internal server error occurs").build();
+        } catch (ApplicationManagementException e) {
+            log.error("Application Management Exception occurs", e);
+            return Response.status(Response.Status.INTERNAL_SERVER_ERROR)
+                    .entity("Application with UUID" + uuid + " Internal server error occurs").build();
+        }
+        return Response.status(Response.Status.OK).entity(ratedUsers).build();
+    }
+
+    @Override
+    @POST
+    @Consumes("uuid/stars/json")
```

Comment:
@NishadiPrasangini parameter in the consumes annotation is wrong. It should be `application/json`

Reply:
This change is done.